spec/controllers/projects_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe ProjectsController, type: :controller do
```

Comment:
I generally prefer Capybara specs over Controller specs, although they take a little bit more time. @santib what's your take on this?

Reply:
Same. I'd go with feature specs https://www.jorgemanrubia.com/2018/05/19/on-rails-testing/

Reply:
As I spoke with @hvilloria we are going to keep this until we have finished the view completely. This is because we have many changes ongoing.

Reply:
Sounds reasonable :+1: